src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -70,6 +73,24 @@ export default class AircraftController {
     }
 
     /**
+     * Add a new `AircraftConflict` instance to the list of existing conflicts
+     *
+     * @for AircraftController
+     * @method addConflict
+     * @param  {Aircraft} aircraft      aircraft 1
+     * @param  {Aircraft} otherAircraft aircraft 2
+     */
+    addConflict(aircraft, otherAircraft) {
+        const conflict = new AircraftConflict(aircraft, otherAircraft);
+
+        if (conflict.isDuplicate() || conflict.shouldBeRemoved()) {
+            return;
```

Comment:
Actually I'm a moron. This doesn't need to check `.isDuplicate`, as that is already done in `.shouldBeRemoved()`...